ballerina/listener.bal

```diff
@@ -86,6 +86,12 @@ public class Listener {
         __Schema & readonly schema = engine.getSchema();
         __Type? subscriptionType = schema.subscriptionType;
         if subscriptionType is __Type && self.wsListener is () {
+            string httpVersion = self.httpListener.getConfig().httpVersion;
+            if httpVersion !is http:HTTP_1_1|http:HTTP_1_0 {
```

Comment:
Shouldn't this be easy if we just check for `HTTP_2` here?

Reply:
Added this because of we surely know that these two versions are supported subscriptions.